troi/filters.py

```diff
@@ -111,3 +112,56 @@ def read(self, inputs, debug=False):
                 results.append(r)
 
         return results
+
+
+class DuplicateRecordingFilterElement(troi.Element):
+    """This Element takes a list of recordings and removes any duplicate recordings
+    that appear directly after each other (based on recording MBID)
+    If recordings are in the order mbid1, mbid2, mbid1, this will not be considered
+    a duplicate.
+    """
```

Comment:
This is a very oddly specific filter -- I think the name of it is currently misleading. Judging by the name I would expect it to filter out all duplicates. Maybe call it ConsecutiveRecordingFilterElement()? Or maybe add the functionality to remove all duplicates while preserving the recording order?

Reply:
Agreed, being able to to filter all items preserving order is a good idea. I guess this is the same behaviour as `uniq`!

Reply:
I think in general Elements/Filters should preserve order -- while we don't rely on that now, I think it is a good goal to shoot for -- I suspect it will be useful later.

Reply:
after discussion, we added 2 filters for each of these behaviours.
Was also missing tests, so added those.